src/alvin0319/OffHand/OffHandInventory.php

```diff
@@ -27,75 +27,57 @@
 
 namespace alvin0319\OffHand;
 
-use BadMethodCallException;
-use pocketmine\entity\Entity;
 use pocketmine\inventory\BaseInventory;
-use pocketmine\item\Item;
 use pocketmine\network\mcpe\protocol\MobEquipmentPacket;
-use pocketmine\network\mcpe\protocol\InventoryContentPacket;
-use pocketmine\network\mcpe\protocol\types\inventory\ItemStackWrapper;
-use pocketmine\Player;
-
-class OffHandInventory extends BaseInventory{
-	public const SLOT_OFFHAND = 0;
-
-	/** @var Player */
+use pocketmine\network\mcpe\protocol\types\ContainerIds;
+use pocketmine\nbt\tag\ListTag;
+use pocketmine\item\Item;
+use pocketmine\entity\Human;
+use pocketmine\Server;
+use pocketmine\utils\MainLogger;
+class OffhandInventory extends BaseInventory{
+    /** @var Human */
 	protected $holder;
-
-	public function __construct(Player $holder){
-		parent::__construct([], 1);
+	public function __construct(Human $holder){
 		$this->holder = $holder;
+		parent::__construct();
```

Comment:
This will cause argument error

Reply:
Well I used that aswell and it worked perfectly fine

Reply:
Basically the order of that doesn’t matter, and isn’t necessary to complete the arguments because the BaseInventory auto completes them, the contents argument is completed as a [] and the size is completed as null, so there will be no default contents and the size will be depend the getDefaultSize method